docs/topics/proc_verifying-pull-requests.adoc

```diff
@@ -24,6 +24,7 @@ If the button says *Unwatch*, you are already subscribed.
 +
 NOTE: If there is no reference to an open issue and the change is not trivial, request that an issue be created.
 +
+.. Link to rendered documentation with changes from pull request is available in pull request as a GitHub check. Open the pull request and Click on `Details` of `deploy-netlify`. 
```

Comment:
Thank you, @pmacko1 the sentence is OK, but please consider adding some detail to it.
I suggest rephrasing it as follows:

> To preview the rendered document with the changes in your pull request:
> *  Navigate to the pull request you want the preview.
> * Select the _Conversation_ tab and Scroll down to the bottom of the screen
> * Click _Show all checks_ to expand the checks section
> * Click _Details_ next to "deploy/netlify"